src/api.js

```diff
@@ -42,6 +42,13 @@ var ping = {
   }
 };
 
+var debug = Debug('api');
+
+/* In production, log authorizations so they are included in papertrail regardless of
+ * DEBUG settings; otherwise, log with debug
+ */
+var authLog = process.env.NODE_ENV === 'production' ? console.log : Debug('api.authz');
```

Comment:
Maybe this is a nit, I know it's not required if I test it in node... but how about:
```js
var authLog = (...args) => console.log(args...);
if process.env.NODE_ENV === 'production') {
  var authLog = Debug('api.authz');
}
```
I'm slightly concern about not binding `console.log` to `console`.
`console.log.bind(console)` also works... Also I like not having it in one line, but that's a nit.